### ISSUE TITLE
[question] bootstrap 4?

### ISSUE THREAD

**Issue by joneldiablo** [rajasegar/PUG-Bootstrap#29]
this project is for pug and bootstrap 4 components?

**Comment by rajasegar** [comment]
@joneldiablo Yes this is for pug but it is not yet updated to support Bootstrap 4

**Comment by lyoko17220** [comment]
So, when Bootstrap 4 ?

**Comment by rajasegar** [comment]
To be honest @lyoko17220 I could not find time to work on it, but after looking at this thread i have made up my mind to kick start the project, will update you all asap once it is launched.
👍

**Comment by zlix12** [comment]
@rajasegar I would be interested in helping with the pug bs4 project.

**Comment by ksaadDE** [comment]
I have the same "issue" here.

So it's 2020 and it still contains outdated Bootstrap. I tried to fix it myself, finding out that a lot of the stuff used in the snippets are already outdated, partly or fully. Sad for the hard work...

maybe a "rewrite from scratch" could be more successful instead of fixing the parts since a lot changed in BS4+

 @rajasegar you wrote somewhere something about you need funding for it. What do we talk about? $100 $1000? :smile:

**Comment by rajasegar** [comment]
@ksaadDE Created a tracking issue for the same
#32 
All the changes are pushed on to the b4 branch https://github.com/rajasegar/PUG-Bootstrap/tree/b4
Will start working on this, but it will take some time, I cannot give a timeline, wish to complete it by this end of year, so hopefully we will have v4 support next year. 
And meanwhile I would also accept funding , this is my patreon page 
https://www.patreon.com/rajasegar

**Comment by ksaadDE** [comment]
> @ksaadDE Created a tracking issue for the same
> #32
> All the changes are pushed on to the b4 branch https://github.com/rajasegar/PUG-Bootstrap/tree/b4
> Will start working on this, but it will take some time, I cannot give a timeline, wish to complete it by this end of year, so hopefully we will have v4 support next year.
> And meanwhile I would also accept funding , this is my patreon page
> https://www.patreon.com/rajasegar

Awesome, sounds great!

I tried to fix (at least) the navbar. The Rest seems to work, but we'll have to change the Navbar functionality. Take a look at my repo and you'll see the diff.  Maybe you can change it to your preferred Style, so we will have the same functionality like before.

Just tell me how much you need over all for completing this project, I know that you have patreon, but that doesn't mean how much you're needing ;-) People are spending more when they know a) what for and b) how much is generally needed.

Greetz

**Comment by joneldiablo** [comment]
and bs5?
XD
